common/build/eslint-config-fluid/minimal-deprecated.js

```diff
@@ -430,6 +434,14 @@ module.exports = {
 				// Disabled for test files
 				"@typescript-eslint/consistent-type-exports": "off",
 				"@typescript-eslint/consistent-type-imports": "off",
+
+				// For test files only, additionally allow import of '/test' exports.
+				"import/no-internal-modules": [
+					"error",
+					{
+						allow: ["@fluidframework/*/test"].concat(permittedImports),
```

Comment:
I am now thinking we should make this more relaxed.
"@fluid*/*/test*" and "@fluid*/*/internal/test*"
Seem good. I do think "test-utils" is a better name than "test" in most cases because we aren't _normally_ testing that package.